### PR TITLE
falco community survey blog

### PR DESCRIPTION
**Pull request by jsalinas29**
images and copy for summary of community survey

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

/kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:
copy editing 

**Which issue(s) this PR fixes**:
new content summary for community survey

Fixes #

**Special notes for your reviewer**:
falco community survey blog 
copy + images
Signed-off-by: jsalinas29 <salinas.jacque@gmail.com>

### CODE REVIEW

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+Security threat detection tops the list as the driving force behind Falco adoption, followed closely by auditing and compliance. Sandbox testing and incident response are also cited as compelling reasons to embrace Falco. Notably, one participant isn't using Falco, while a couple of others leverage its libraries and policy language within their unique open-source projects. These findings highlight Falco's versatility in catering to diverse security requirements across numerous domains.
+
+**2. Why are you adopting Falco?**
+* Majority adopted Falco for security threat detection (58.3%)
+* Auditing and compliance were common reasons (45.8% each)
+* Sandbox testing and incident response also mentioned
+* Unique use cases: integrating Falco libraries into projects
+* Growing popularity as versatile security solution across fields
+
+![](https://i.imgur.com/JggZe0E.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:43]:
```suggestion
content/en/blog/community survey 2023/reasons for adopting falco.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+**3. Which teams use Falco at your company?**
+* Security teams most common users (45.8%)
+* DevOps teams also prominent (41.7%)
+* Platform engineering, R&D, agent developers, and early-adopting solution architects mentioned
+* 25% reported no Falco usage in their company
+* Some teams utilize Falco libraries directly
+
+![](https://i.imgur.com/8Ch67fN.png)
+
+
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:58]:
```suggestion
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+Falco predominantly finds a home within security teams, though DevOps squads are also known to harness its capabilities. Other adopters include platform engineering units, R&D departments, agent developers, and solution architects in the early stages of adoption. Interestingly, some survey participants don't use Falco at all, while others opt for its libraries directly, bypassing the full solution to meet their unique needs.
+
+**3. Which teams use Falco at your company?**
+* Security teams most common users (45.8%)
+* DevOps teams also prominent (41.7%)
+* Platform engineering, R&D, agent developers, and early-adopting solution architects mentioned
+* 25% reported no Falco usage in their company
+* Some teams utilize Falco libraries directly
+
+![](https://i.imgur.com/8Ch67fN.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:56]:
```suggestion
content/en/blog/community survey 2023/teams that use falco.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+Most Falco enthusiasts opt for the official Helm chart when it comes to deployment, but the Falco community is nothing if not creative. Some users prefer official container images, bespoke manifest files, or even official packages like .deb, .rpm, and .tar.gz. Meanwhile, others have ventured off the beaten path, experimenting with custom shell script wrappers atop Helm, homegrown repositories for source compilation, tailor-made Helm charts combined with custom-built container images, or integrating Falco libraries directly into their agent and kernel modules. The sky's the limit when it comes to deploying Falco!
+
+**4. How are you deploying Falco?**
+* Majority using official Helm chart (62.5%)
+* Official container images (20.8%)
+* Custom manifest files (16.7%)
+* Official packages (.deb/.rpm/.tar.gz) (12.5%)
+* Various alternative methods (16.7%)
+
+![](https://i.imgur.com/LRthsCS.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:69]:
```suggestion
content/en/blog/community survey 2023/deploying falco.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+**Rulesets**
+Most users rely on the default ruleset when using Falco, while a significant number of users prefer to maintain their own ruleset to tailor it to their specific needs. Others choose to take advantage of the new ruleset feature to further enhance their security. The flexibility and adaptability of Falco's ruleset options provide users with a customized and comprehensive approach to their security needs.
+
+**5**. **What ruleset do you use with Falco?** 
+* Default ruleset used by 54.2%
+* 50% maintain their own ruleset
+* New ruleset feature utilized by 29.2%
+* Other methods account for 4.2%
+
+![](https://i.imgur.com/8WiWHIo.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:80]:
```suggestion
content/en/blog/community survey 2023/rulesets.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+**6. What challenges have you faced with adopting Falco?**
+* The top two challenges faced by adopters of Falco were kernel module/eBPF probe installations (41.7%) and management of rules (41.7%)
+* Integrations with third parties were also identified as a challenge by 37.5% of the respondents
+* Tuning, management of plugins, and performance were also mentioned as challenges by a significant number of participants (33.3%, 29.2%, and 16.7%, respectively)
+* Some respondents (12.5%) indicated that they do not use Falco, while others mentioned other challenges, such as integrating Falco security event logs with AWS CloudWatch and using AWS FireLens and Fluent Bit log event forwarding.
+
+![](https://i.imgur.com/1k5oSka.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:91]:
```suggestion
content/en/blog/community survey 2023/challenges adopting.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+The survey asked participants to rate Falco's documentation on a scale from 1 to 5, with 5 being "very helpful" and 1 being "not helpful." All 24 participants responded to this question, and the average rating for Falco's documentation was 3.7. While this is a solid rating, it indicates that there is room for improvement. Some participants rated Falco's documentation as highly helpful (with ratings of 4 or 5), while others found it to be less helpful (with ratings of 1 or 2). The survey data highlights the importance of clear and concise documentation to assist users in getting the most out of Falco.
+
+**8. On a scale of 1 - 5, how would you rate Falco’s documentation?**
+* The average rating was 3.7
+* 41.7% of respondents rated the documentation 2 out of 5
+* 29.2% of respondents rated the documentation 3 out of 5
+* 25% of respondents rated the documentation 4 out of 5
+* 4.2% of respondents rated the documentation 1 out of 5
+
+![](https://i.imgur.com/EjXJFQC.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:128]:
```suggestion
content/en/blog/community survey 2023/documentation rating.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+
+**10. On a scale of 1 - 5, how would you rate Falco’s community support?**
+* All 24 respondents rated Falco's community support on a scale of 1-5, with an average rating of 4.2
+* Over 45% of respondents rated the community support as "Very helpful" (5), while 37.5% rated it as "Helpful" (4)
+* Only one respondent rated the community support as "Not helpful" (1)
+* The results indicate that Falco has a strong and supportive community, with the majority of respondents finding the support helpful or very helpful
+
+![](https://i.imgur.com/sXlJRd5.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:151]:
```suggestion
content/en/blog/community survey 2023/community support.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+**Contributing to Falco**
+The results highlight a mix of engagement levels within the community and potential for increased involvement from those who would like to contribute.
+
+**11. Are you an active contributor to the project?**
+* Out of 24 respondents, 33.3% were active contributors to the Falco project
+* An equal percentage of respondents (33.3%) expressed interest in becoming contributors in the future
+* 33.3% of respondents were not active contributors to the project
+
+![](https://i.imgur.com/nlTzkVA.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:161]:
```suggestion
content/en/blog/community survey 2023/contributor.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+
+**12. On a scale of 1 - 5, how satisfied are you with the communication of the Falco project?**
+* The survey asked participants to rate their satisfaction with the communication of the Falco project on a scale of 1-5
+* The average rating of the Falco project's communication was 4.2
+* A majority of respondents rated Falco's communication as either highly satisfactory (37.5%) or satisfactory (50%)
+* Only a small percentage of respondents were either neutral or not satisfied with Falco's communication (8.3% and 4.2% respectively)
+* These results demonstrate that overall, respondents are satisfied with the communication of the Falco project
+
+![](https://i.imgur.com/B2oBqXs.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:173]:
```suggestion
content/en/blog/community survey 2023/communication satisfaction.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+**Weekly community calls**
+Falco's community calls are an essential part of its communication strategy, allowing users to connect and collaborate with other members of the community. The survey asked participants whether they attend these calls, with 24 out of 24 respondents answering. The results showed that a majority of respondents do not attend the weekly community calls, with an almost equal number of respondents indicating they are planning to attend in the future. A smaller number of respondents attend the calls from time to time, and only a few respondents attend almost every week. These results highlight the importance of ensuring that community calls are accessible and convenient for all members, and that alternative ways of communication are available for those unable to attend.
+
+**13. Do you attend the weekly community calls?**
+* 29.2% of respondents do not attend the calls
+* Another 29.2% are not attending, but plan to in the future
+* Another 29.2% attend the calls from time to time
+* Only 12.5% of respondents attend the weekly community calls almost every week
+
+![](https://i.imgur.com/jbHqSrq.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:184]:
```suggestion
content/en/blog/community survey 2023/community call.png
```

File: content/en/blog/community survey 2023/Falco community survey results 2023 (1).md [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+
+**14. Are you interested in sharing your Falco adoption story and publishing the your company's name in Falco's adopters file?**
+* 54.2% of respondents were not interested in sharing their Falco adoption story, and preferred that it not be shared at all.
+* 25% of respondents were willing to share their story, but only privately with the maintainers.
+* 20.8% of respondents were willing to share their story publicly.
+* All 24 respondents answered this question.
+
+![](https://i.imgur.com/jIwoeci.png)
```
Comment (jsalinas29) [inline content/en/blog/community survey 2023/Falco community survey results 2023 (1).md:195]:
```suggestion
content/en/blog/community survey 2023/adopters.png
```

**Review by jsalinas29 — COMMENTED** [review]
referencing the correct images
removing a space

**Review by vjjmiras — APPROVED** [review]
LGTM

**Pull request merged**